### ISSUE TITLE
shoutdown process error with python 3.4 and pyqt/PySide

### ISSUE THREAD

**Issue by b0051fef-975b-4ba7-a1b8-e4bcbb6c1769** [python/cpython#74984]
BPO | [30801](https://bugs.python.org/issue30801)
--- | :---
Nosy | @terryjreedy, @larryhastings, @alfonnews
PRs | <li>python/cpython#2413</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2017-07-11.12:49:41.006>
created_at = <Date 2017-06-29.07:56:43.037>
labels = ['library', 'type-crash']
title = 'shoutdown process error with python 3.4 and pyqt/PySide'
updated_at = <Date 2017-07-24.22:44:41.158>
user = 'https://github.com/alfonnews'
```

bugs.python.org fields:
```python
activity = <Date 2017-07-24.22:44:41.158>
actor = 'terry.reedy'
assignee = 'none'
closed = True
closed_date = <Date 2017-07-11.12:49:41.006>
closer = 'larry'
components = ['Library (Lib)']
creation = <Date 2017-06-29.07:56:43.037>
creator = 'alberfontan1'
dependencies = []
files = []
hgrepos = []
issue_num = 30801
keywords = []
message_count = 3.0
messages = ['297261', '298154', '299027']
nosy_count = 3.0
nosy_names = ['terry.reedy', 'larry', 'alberfontan1']
pr_nums = ['2413']
priority = 'normal'
resolution = 'wont fix'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'crash'
url = 'https://bugs.python.org/issue30801'
versions = ['Python 3.4']
```

</p></details>

**Comment by b0051fef-975b-4ba7-a1b8-e4bcbb6c1769** [comment]
For a package applicaction with pyqt/PySide, Python 3.4 on Windows, appers a window fatal error when shoutdown process manager of multiprocessing

**Comment by larryhastings** [comment]
Python 3.4 no longer accepts bug fixes; it is in "security fixes only" mode.  Since this is not a security fix, it will not be accepted into Python 3.4.

If this bug affects other versions of Python, please file a new bug.  Although, unless you move \*very* quickly, this fix won't be accepted into 3.5 either, as it's going to move into "security fixes only" mode later this month.

Finally, your submitted patch ([PR 2413](https://github.com/python/cpython/pull/2413)) does not apply to the 3.4 branch, it applies to master.  I have no opinion about whether or not this code should be accepted into master.  But the PR talks about fixing a bug in 3.4, which is wrong.

**Comment by terryjreedy** [comment]
You must demonstrate that there is a problem with 3.6 or 3.7 and then write a patch that fixes the problem in those versions.
